datastore/db/management/commands/load_data_package.py

```diff
@@ -54,6 +58,11 @@ def load_data(self):
                             getter_run=getter_run,
                         )
                     )
+                else:
+                    logger.warning(
+                        "Found unacceptable data in grant '%s'",
+                        json.dumps(grant.get("id")),
```

Comment:
What's the need for `json.dumps` here?  Could we do 
```python
logging.warn(f"...unacceptable ... {grant['id']}")
```
?

Reply:
To hedge against the grant id containing newlines, or potentially any other random unicode or otherwise character, this was my quick-n-lazy way of escaping everything to not break the log output. Probably there's a better way of escaping?
That's a point, i should at least add a comment explaining, it is weird

Reply:
I couldn't remember if f-strings escape expression output, but apparantly not:
```python
>>> w = "wor\nld"
>>> print(f"hello {w}")
hello wor
ld
```